.github/workflows/publish-commit-bottles.yml

```diff
@@ -107,11 +107,36 @@ jobs:
           GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
         working-directory: ${{steps.set-up-homebrew.outputs.repository-path}}
 
+      - name: Get current branch and remote
+        id: branch-and-remote-info
+        working-directory: ${{steps.set-up-homebrew.outputs.repository-path}}
+        run: |
+          branch="$(git branch --show-current)"
+          if [[ -n "$branch" ]]
+          then
+            echo "branch=$branch" >> "$GITHUB_OUTPUT"
+          else
+            echo "::error ::Not on a branch!"
+            exit 1
+          fi
```

Comment:
This has an unhandled edge-case.

If the PR branch is also called `master` (or otherwise conflicts with a branch name in the homebrew/core repo on the runner), then `git branch --show-current` will return `$username/master`, which will not work for the `git-try-push` action below.